pyproject.toml

```diff
@@ -76,7 +76,7 @@ filelock = "^3.8.0"
 checksumdir = "^1.2.0"
 
 [tool.poetry.extras]
-pydantic = ["pydantic", "email-validator"]
+extras = ["pydantic", "email-validator", "aws-xray-sdk", "fastjsonschema"]
```

Comment:
Can we rename it to `all` and also have a group per each feature e.g. `tracer`, `parser`, etc.?